seed/seed20.go

```diff
@@ -82,6 +84,82 @@ type seed20 struct {
 	essentialSnapsNum int
 }
 
+// Copy implement Copier interface. Copies the seed to the given root, where
+// root is treated as the root of a seed partition.
+func (s *seed20) Copy(root string) (err error) {
+	// TODO: right now this method requires you to call LoadAssertions and
+	// LoadMeta first. is that okay? should Copier embed the Seed interface to
+	// make that easier? i could call them in here, but then i'd need to take in
+	// a db as a parameter.
```

Comment:
this is fine I think but the doc comment on the Copier interface should explain this

Reply:
In that case, should `Copier` embed `Seed`? Since `Copier` will requirer methods from `Seed`?

Reply:
Comment is updated.

Reply:
yes, including Seed is fine, if you look other interfaces do that too